spec/models/guide_spec.rb

```diff
@@ -4,6 +4,8 @@
   let!(:extra_user) { create(:user, first_name: 'Ignatius', last_name: 'Reilly') }
   let(:guide) { create(:guide) }
 
+  it { expect(Guide.new.to_resource_h.keys).to eq Mumuki::Domain::Store::Github::GuideSchema.fields.map(&:name) }
```

Comment:
@felipecalvo check this

Reply:
nice, so we can make sure that everything's up to date, right?

Reply:
Right